client/src/Components/QuestionsContainer/QuestionsList/QuestionsList.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from "react"
+import { ListGroup, Button } from "react-bootstrap"
+import QuestionListEntry from "./QuestionsListEntry/QuestionsListEntry.jsx"
+import QuestionModal from "./QuestionModal/QuestionModal.jsx"
+import useQA from "../../../contexts/QAContext.jsx"
+import styled from "styled-components"
+import InfiniteScroll from "react-infinite-scroll-component"
+
+const Button1 = styled.button`
+  margin-right: 15px;
+  border-radius: 0px;
+  border: 2px solid black;
+  background-color: white;
+  color: black;
+  padding: 14px 28px;
+  font-size: 16px;
+  cursor: pointer;
+`
+
+const QuestionsList = (props) => {
+  // accepts list of questions: load up to four questions
+  // in return statement will map over list of questions in component QuestionsListEntry
+  // FOR NOW will contain the "more answered Questions" and "Add a Question" button underneath
+  const [questionListData, setQuestionListData] = useState([])
+  const [renderQuestionsListData, setRenderQuestionsListData] = useState([])
+  const [questionsCount, setQuestionsCount] = useState(4)
+  const [hasMore, setHasMore] = useState(true)
+
+  const context = useQA()
+
+  useEffect(() => {
+    // collecting entire Question List for a certain product ID
+    getData()
+  }, [])
+
+  useEffect(() => {
+    // sets the questions to be rendered; as questionsCount increases the more questions that are rendered
+    getRenderData()
+  }, [questionsCount, questionListData])
+
+  useEffect(() => {
+    // filters the questions by the value being searched in the search bar
+    filterData()
+  }, [props.filterBySearch])
+
+  const getData = () => {
```

Comment:
would it be better to store questionDataList in `QuestionList.jsx`, or maybe inside of `QAContext` where it does the fetching?